### PR TITLE
Mobile emulation for Chrome browser

### PR DESCRIPTION
**Pull request by nirmalashish**

### CODE REVIEW

**Comment by anandbagmar** [comment]
@nirmalashish - a few suggestions:
1. Can we add a new test for mobile web to show how this will be used?
2. Update the README

**Comment by anandbagmar** [comment]
@nirmalashish - yes, I understand the logic is in the downstream. Hence a working test to show how it can be used will be helpful.

**Comment by nirmalashish** [comment]
@anandbagmar 
- added a new feature file that tells user how to pass mobile emulation specific parameters; also added guidance at step def level
- updated README for this new feature

File: README.md [hunk 1]
```diff
@@ -139,6 +139,18 @@ You can run these tests as below:
...
+
+The framework now supports launching web browsers in Mobile Emulation View.
+Instead of passing simple browser names in steps like 'chrome', 'firefox', etc, 
+you can pass more informative parameters like 'chrome-mobile1', 'safari-mobile2', etc
+
+These informative parameters refer to Browser Config json file for actual device names
+
+Example:
+
+![mobile-emulation-on-web-browser.png](mobile-emulation-on-web-browser.png)
```
Comment (anandbagmar) [inline README.md:152]: this png seems to be missed for checking

File: src/main/resources/default_browser_config.json [hunk 2]
```diff
@@ -51,11 +55,19 @@
     "verboseLogging": true,
     "headlessOptions": {
       "headless": true
-    }
+    },
+    "tab1" : "iPad",
```
Comment (anandbagmar) [inline src/main/resources/default_browser_config.json:59]: though we have added the values for firefox and safari, this is not yet implemented in Drivers, right?

Comment (nirmalashish) [reply src/main/resources/default_browser_config.json:59]: yes, the logic is only present for Chrome as of now

File: README.md (hunk 1 above)
Comment (nirmalashish) [reply README.md:152]: added now (previously ignored due to gitignore)

**Pull request merged**